Procfile

```
@@ -1,2 +1,3 @@
-web: vendor/bin/heroku-php-nginx -C nginx.conf public/
-release: if [ -n "$DB_HOST" ]; then php artisan migrate --force; fi
+web: composer warmup && $(composer config bin-dir)/heroku-php-nginx -C nginx.conf public/
+scheduler: php -d memory_limit=512M artisan queue:work --stop-when-empty
+release: php artisan migrate --force && php artisan cache:clear
```

```suggestion
release: php artisan migrate --force && php artisan cache:clear || echo "unable to complete release"
```

tested with this `rm /dev/zero && rm /dev/zero || echo "I failed, but succeeded"`